src/olympia/lib/settings_base.py

```diff
@@ -1767,8 +1767,9 @@ def get_raven_release():
 
     if os.path.exists(version_json):
         try:
-            with open(version_json, 'rb') as fobj:
-                data = json.loads(fobj.read())
+            with open(version_json) as fobj:
```

Comment:
explicit is always better than implicit, let's set `'r'` here too. I think there are not many occurrences in our code where we don't set it.

Reply:
Fixed in https://github.com/mozilla/addons-server/pull/10414/commits/4633b7a63d2100b3816911c610f178f681c96b9d